pop-api/examples/fungibles/utils.rs

```diff
@@ -0,0 +1,206 @@
+// A set of helper methods to test the contract calls.
```

Comment:
Should be in the folder of tests. Now it can be confused being utils for the contract.